crypto/cipher_extra/e_chacha20poly1305.c

```diff
@@ -78,11 +114,11 @@ static void poly1305_update_length(poly1305_state *poly1305, size_t data_len) {
 
 // calc_tag fills |tag| with the authentication tag for the given inputs.
 static void calc_tag(uint8_t tag[POLY1305_TAG_LEN], const uint8_t *key,
-                     const uint8_t nonce[12], const uint8_t *ad, size_t ad_len,
-                     const uint8_t *ciphertext, size_t ciphertext_len,
-                     const uint8_t *ciphertext_extra,
+                     const uint8_t nonce[CHACHA_IV_LEN], const uint8_t *ad,
+                     size_t ad_len, const uint8_t *ciphertext,
+                     size_t ciphertext_len, const uint8_t *ciphertext_extra,
```

Comment:
For future PRs please separate simple refactors like this into a separate PR, it makes it much easier to review.